pkg/apis/batch/v1alpha1/job.go

```diff
@@ -265,6 +269,10 @@ type JobStatus struct {
 	// +optional
 	RetryCount int32 `json:"retryCount,omitempty" protobuf:"bytes,10,opt,name=retryCount"`
 
+	// The job running duration
```

Comment:
Add more comments, should be clear this the duration from the running to the complete.